ibis/impala/compiler.py

```diff
@@ -39,18 +45,18 @@ def to_sql(expr, context=None):
 # Select compilation
 
 
-class ImpalaSelectBuilder(comp.SelectBuilder):
+class ImpalaSelectBuilder(BaseSelectBuilder):
     @property
     def _select_class(self):
         return ImpalaSelect
 
 
-class ImpalaQueryBuilder(comp.QueryBuilder):
+class ImpalaQueryBuilder(BaseQueryBuilder):
 
     select_builder = ImpalaSelectBuilder
 
 
-class ImpalaContext(comp.QueryContext):
+class ImpalaContext(BaseContext):
     def _to_sql(self, expr, ctx):
         return to_sql(expr, ctx)
```

Comment:
I think `_to_sql` can be removed, since it's the same as the base class.

As with dialect, not sure where this `ImpalaContext` is used, but maybe we can just get rid of it and use `BaseContext` instead.

Reply:
This i was able to just replace with just BaseContext